crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -90,6 +90,11 @@ impl SessionType {
     }
 }
 
+/// A struct witnessing a successful decryption of an Olm-encrypted to-device
+/// event.
+///
+/// Contains the decrypted event plaintext along with some associated metadata,
+/// such as the identity (Curve25519) key of the to-device event sender.
 #[derive(Debug, Clone)]
 pub struct OlmDecryptionInfo {
```

Comment:
Should users be able to create an instance of this struct? If no, we should probably make it `#[non_exhaustive]` to forbid that.

Reply:
This should really be `pub(crate)` instead. Fixed.